docker/test-app.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
```

Comment:
1. What are the main goals for this script? Why can't we use AMT, for example?

2. Let's not put scripts in the root of docker directory structure — it's already becoming too large and requires to be reorganized.

We have `docker/_scripts` and `docker/ssl`, I'd suggest to:
- rename `docker/_scripts` → `docker/scripts` (please don't forget to change paths inside scripts)
- create the `docker/tests` subdirectory and put the file there

Reply:
AMT is a little "overpowered' for that. this script is like simple little client application  that make read write requests to DB with defined encryption config